.travis.yml

```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```

Comment:
I've been using the new beta versions to test features across repos and I've been checking the GitHub releases/tags to know which versions are available. This has been useful to me personally, but I'm also not sure if there is a better way to find the latest beta version for a repo.